content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -98,6 +98,9 @@ is local to a namespace.  This is useful for using the same configuration across
 multiple namespaces such as Development, Staging and Production.  If you want to reach
 across namespaces, you need to use the fully qualified domain name (FQDN).
 
+As a result, all namespace names must be valid
+[RFC 1123 DNS labels](/docs/concepts/overview/working-with-objects/names/#dns-label-names).
```

Comment:
Yes but no. Nearly all [object names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) must be valid RFC 1123 DNS labels (it doesn't just apply to namespaces).

For that reason I'd mention this detail earlier in the page, in case people think the use of DNS is the reason why. Also, I'd link to https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names

If you're willing to, @adrianludwin, consult the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) and revise the whole page to match it.

Reply:
I thought that most object names needed to be DNS _subdomains_, not labels. E.g. I was able to create a deployment, pod, RBAC role, network policy, and secret with dots in their names. By contrast, services (whose names must fit _into_ a DNS subdomain) and namespaces (ditto) are limited to DNS labels. There may be other places where we're limited to DNS labels without needing to fit into a subdomain but I don't know what they are.

I don't really have time to revise the entire page, was there a specific error in my addition that violates the style guide?

Thanks!

Reply:
The page needs revising, so if someone has time to we'd appreciate it.